README.md

```diff
@@ -322,6 +324,8 @@ The default port is 1337, to use a different port set the PORT environment varia
 $ PORT=8080 parse-server --appId APPLICATION_ID --masterKey MASTER_KEY
 ```
 
+Must set `PARSE_PUBLIC_SERVER_URL` equal to `PARSE_SERVER_URL` if using HTTPS ([details here](https://github.com/ParsePlatform/parse-server/issues/2743)).
```

Comment:
That's not true.

serverURL should **always** point to the localhost server (ex: `http://localhost:1234/parse`). IFF the server firewall prevents loopback connections (from localhost to localhost) it should be set tu a publicly available URL.

publicServerURL should **always** point to a publicly accessible URL, with a FQDN, preferably over HTTPS.